src/adapters/justpremium.js

```diff
@@ -0,0 +1,245 @@
+import bidfactory from 'src/bidfactory';
+import bidmanager from 'src/bidmanager';
+import adloader from 'src/adloader';
+import * as utils from 'src/utils';
+
+const JustpremiumAdapter = function JustpremiumAdapter() {
+  const top = window.top;
+  const d = top.document;
+
+  function arrayUnique(array) {
+    var a = array.concat();
+    for (var i = 0; i < a.length; ++i) {
+      for (var j = i + 1; j < a.length; ++j) {
+        if (a[i] === a[j]) {
+          a.splice(j--, 1);
+        }
+      }
+    }
+
+    return a;
+  }
+
+  function readCookie(name) {
+    const nameEQ = name + '=';
+    const ca = document.cookie.split(';');
+    for (let i = 0; i < ca.length; i++) {
+      var c = ca[i];
+      while (c.charAt(0) == ' ') c = c.substring(1, c.length);
+      if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length, c.length);
+    }
+    return null;
+  }
+
+  let bids;
+  let cookieLoaded = false;
+  let adManagerLoaded = false;
+  let jPAM = top.jPAM = top.jPAM || window.jPAM || {};
+
+  const dConfig = jPAM._dev ||
+    {
+      toLoad: null,
+      server: null
+    };
+  const libVer = readCookie('jpxhbadp') || null;
+  const toLoad = dConfig.toLoad || [d.location.protocol + '//d2nvliyzbo36lk.cloudfront.net/adp/' + (libVer ? libVer + '/' : '') + 'bc.js'];
+  const server = dConfig.server || d.location.protocol + '//pre.ads.justpremium.com/v/1.4';
+
+  function loadCookie() {
+    if (cookieLoaded) return;
+    cookieLoaded = true;
+    adloader.loadScript(d.location.protocol + '//ox-d.justpremium.com/w/1.0/cj');
+  }
+
+  function loadTag(params, callback) {
+    const keys = Object.keys(params || {});
+    const url = `${server}${keys.length ? '/?' : ''}${keys.map(key => `${key}=${params[key]}`).join('&')}`;
+    adloader.loadScript(url, callback);
+  }
+
+  function onLoad() {
+    jPAM = top.jPAM = Jpx.JAM.instance({
+      plugins: ['bidder']
+    });
+  }
+
+  function loadResources() {
+    if (toLoad.length > 0) {
```

Comment:
In development mode, instead of one file, few separated ones are loaded. For this purpose I would prefer to leave it as it is. Of course if its ok for you.